runtime.md

```diff
@@ -129,12 +129,16 @@ This operation MUST run the user-specified program as specified by [`process`](c
 This operation MUST generate an error if `process` was not set.
 
 ### <a name="runtimeKill" />Kill
-`kill <container-id> <signal>`
+`kill [OPTION] <container-id> <signal>`
 
 This operation MUST [generate an error](#errors) if it is not provided the container ID.
 Attempting to send a signal to a container that is neither [`created` nor `running`](#state) MUST have no effect on the container and MUST [generate an error](#errors).
 This operation MUST send the specified signal to the container process.
 
+#### OPTION
+
+- `--all` Kill all the processes in the container.
+
```

Comment:
Probably this should be `### <a name="runtimeKillAll" />KillAll`

Reply:
I have updated it. Thanks.